### ISSUE TITLE
A simple trigonometric mathematic question

### ISSUE THREAD

**Issue by ResetPower** [ResetPower/ResetPower#6]
## Question

Given $f(x)=A\tan(\omega x+\phi)$ where $A>0, \omega>0, |\phi|<\frac\pi2$, with nearest zeroes $\frac\pi6$ and $\frac{5\pi}6$, passes through $(0, -3)$, find $f(x)$.

## Solution

The minimum positive period $T=\frac\pi{|\omega|}=\frac{6\pi}5-\frac\pi5=\frac{2\pi}3$.

Since $\omega>0$, $\omega=\frac32$.

Since $f(\frac\pi6)=0$, $A\tan(\frac32\cdot\frac\pi6+\phi)=0$,
which is $\tan(\frac\pi4+\phi)=0$,
which is $\frac\pi4+\phi=k\pi$.

Since $|\phi|<\frac\pi2$, $\phi=-\frac\pi4$.

Since $f(0)=-3$, $A\tan(-\frac\pi4)=-3$,
which is $A=3$.

In conclusion, $f(x)=3\tan(\frac32+\frac\pi4)$.
